wordpress-mobile/WordPress-iOS
# Editor Navigation Bar - Updates the close button icon

This PR updates the Editor's close icon in the navigation bar to use `chevron-left` instead of `chevron-down`.

To test:

- Open the editor
- **Expect** to see the updated close icon in the navigation bar

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
It replaces an existing icon only being used in the editor (it was just added)

PR submission checklist:


UI Changes testing checklist:

## Screenshots

iPhone Light mode | iPhone Dark mode
-|-
<kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/513aa4b2-3461-4c30-9ecb-68401057a211" width="250" /></kbd> | <kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/a7b832b4-9c9c-4f4f-805d-ebe5b0d078f7" width="250" /></kbd>

iPhone Landscape Light mode | iPhone Landscape Dark mode
-|-
<kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/ae1e05c5-2cf7-45fb-96f1-ea3bd8ef70d0" width="250" /></kbd> | <kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/9427bdc2-678c-45c4-b287-b78849857cc3" width="250" /></kbd>

iPhone RTL |
-|
<kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/0d82ef5f-ae54-441a-8260-c22495824ff7" width="250" /></kbd> | 

iPad Light mode | iPad Dark mode
-|-
<kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/69eb47c4-1f45-4726-b851-6dc9e986c9dc" width="250" /></kbd> | <kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/eda60906-bd81-4b02-ad84-799b2b227145" width="250" /></kbd>

iPad Landscape Light mode | iPad Landscape Dark mode
-|-
<kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/25195b8c-9363-4a1f-99a0-2cb0ac74788f" width="250" /></kbd> | <kbd><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4885740/b4f567f2-20ce-48f0-9640-dc5fc2d817ff" width="250" /></kbd>
